variables.tf

```diff
@@ -4,37 +4,38 @@ variable "create_vpc_flow_log" {
 }
 
 variable "log_destination_type" {
-  description = "Controls whether to create the VPC Flow Log with a CloudWatch Logs or S3 bucket destination"
-  type        = "string"
+  description = "Controls whether to create the VPC Flow Log with a `cloud-watch-logs` or `s3` bucket destination"
+  type        = string
   default     = ""
```

Comment:
Ought to be able to `null` all these empty strings also...

Reply:
Not able to null the vpc_id without changing the locals given that one `iam_role_name` uses the variable as part of its naming scheme - "flow-log-${var.vpc_id}"